### DISCUSSION TITLE
How to get value of a dynamically created select

### DISCUSSION THREAD

**Discussion by andresdellachiesa** [react-hook-form/react-hook-form#2537] [Q&A]
Hello everybody. Need some help with the following:

Ok, so, I have four selects. 
Each one them depends on the value of the last one. 

I don't have any problems getting the second select options as I'm actually watching the first select, but here's the thing:

I need to populate all of the selects just by interacting with the first of them. As I get the second select populated, I need the third to do the same. The problem is that I get no value from the second select as I'm not interacting with it, so the "watch" element returns empty.

Could you please help me with this?

Thanks in advance.

**Comment by bluebill1049** [comment]
have you looked at https://react-hook-form.com/api/#useFieldArray? maybe that will help.
